cardano-cli/src/Cardano/CLI/Run/Key.hs

```diff
@@ -205,6 +207,19 @@ runNonExtendedKeyCmd
         writeLazyByteStringFile vkf' $
           textEnvelopeToJSON descr vk
 
+runGenerateMnemonicCmd :: Cmd.KeyGenerateMnemonicCmdArgs -> ExceptT KeyCmdError IO ()
+runGenerateMnemonicCmd
+  Cmd.KeyGenerateMnemonicCmdArgs
+    { mnemonicOutputFormat
+    , mnemonicWords
+    } = do
+    mnemonic <- firstExceptT KeyCmdMnemonicError $ generateMnemonic mnemonicWords
```

Comment:
An extra validation step where we confirm the correct number of words have been generated also can't hurt here.

Reply:
Addressed here: https://github.com/IntersectMBO/cardano-cli/pull/975/commits/b05b69b37552718d90b0e3a6f139bb8dd70f0bae